data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditSuccessMessage.java

```diff
@@ -33,33 +30,35 @@
 /**
  * Represents information for a successful processing of a request which is forwarded to the audit-service.
  * Note there are three classes that effectively represent the same data but represent a different stage of the process.
- * uk.gov.gchq.palisade.service.audit.request.AuditSuccessMessage is the message received by the Audit Service.
- * uk.gov.gchq.palisade.service.results.request.AuditSuccessMessage is the message sent by the results-service.
- * uk.gov.gchq.palisade.service.data.request.AuditSuccessMessage is the message sent by the data-service.
+ * uk.gov.gchq.palisade.service.audit.model.AuditSuccessMessage is the message received by the Audit Service.
+ * uk.gov.gchq.palisade.service.filteredresource.model.AuditSuccessMessage is the message sent by the Filtered Resource Service.
+ * uk.gov.gchq.palisade.service.data.model.AuditSuccessMessage is the message sent by the Data Service.
+ * The one version produced by the data-service is unique in that it includes the leafResourceId, token and the
+ * two counters for the number of records process and records returned as part of the message.
  */
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 public final class AuditSuccessMessage extends AuditMessage {
 
     public static final String RECORDS_PROCESSED = "RECORDS_PROCESSED";
     public static final String RECORDS_RETURNED = "RECORDS_RETURNED";
 
-    private static final ObjectMapper MAPPER = new ObjectMapper();
-
     @JsonProperty("leafResourceId")
     private final String leafResourceId;  //leafResource ID for the resource
 
     @JsonCreator
     private AuditSuccessMessage(
+            final @JsonProperty("leafResourceId") String leafResourceId,
+            final @JsonProperty("token") String token,
```

Comment:
The Token should not be part of this object, 
Also revert the change in ordering, userId should be first like in the other services

Reply:
See previous response